### DISCUSSION TITLE
@Angular 16 -  issue which is fixed for latest version 17 needs to be updated for version 16.9.1 as well

### DISCUSSION THREAD

**Discussion by ps-2512** [primefaces/community#2001] [PrimeNG]
Link to the issue: https://github.com/primefaces/primeng/pull/14527

**Comment by sumit-piq** [comment]
@jepsar It's a blocker issue that breaks UI while updating table data dynamically. Can we please prioritize this fix?

**Reply by jepsar** [reply]
@sumit-piq I'm not a PrimeNG developer. Sorry.

**Reply by sumit-piq** [reply]
Okay. Thanks for moving it under the correct channel.
